scripts/deploy/deployThirdPartyRegistryUpgrade.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers, upgrades } from 'hardhat'
```

Comment:
We can call it `updateThirdPartyRegistryUpgrade.ts`

Reply:
We could remove the `Upgrade` part so it end up as `updateThirdPartyRegistry.ts`